Instsall_Dockerfile.twig

```diff
@@ -66,7 +66,7 @@ COPY debug_config.php /shopware/config.php.dist
 
 {% if consentManagerRequired %}
     RUN /bin/bash -c "/usr/bin/mysqld_safe --skip-grant-tables &" \
-    && sleep 7 \
+    && sleep 10 \
```

Comment:
is there a better alternative than a hard coded sleep?
maybe a while that runs as long as mysql is not ready